### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by larbibaraka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ This is a repo to practice pull request, and contributing to projects, on Github
 You have just to fork this repo to add your name, and maybe link to your GitHub account.
 
 * Mahfoudh Arous [@aladin002dz](https://github.com/aladin002dz)
+* BARAKA LARBI [@larbibaraka](https://github.com/larbibaraka)
```
Comment (aladin002dz) [inline README.md:6]: thanks for your awesome PR, but just to remind you, it is not best practice to send PR from master branch. it is better to create a branch for the changes, and keep the master branch to update the local repo with the command "git pull upstream".

**Review by aladin002dz — COMMENTED** [review]
thanks for this awesome PR, but I am just practicing review and remind you of a best practice in PR (a branch for change, and pull upstream on the master branch before pushing, see: https://aladinstudio.com/2020/07/23/quick-introduction-to-contributing-to-open-source-on-github/)

**Comment by larbibaraka** [comment]
xD ok sorry i thought this repo is to do PRs on

**Comment by aladin002dz** [comment]
> xD ok sorry i thought this repo is to do PRs on

it is, and it was cool to see your contribution. I just suggested that u use the best practice

**Pull request closed without merging**